frontend/src/components/editor-page/sidebar/specific-sidebar-entries/revisions-sidebar-entry/revisions-modal/delete-revision-modal.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * SPDX-FileCopyrightText: 2023 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { deleteRevisionsForNote } from '../../../../../../api/revisions'
+import { useApplicationState } from '../../../../../../hooks/common/use-application-state'
+import { cypressId } from '../../../../../../utils/cypress-attribute'
+import type { ModalVisibilityProps } from '../../../../../common/modals/common-modal'
+import { CommonModal } from '../../../../../common/modals/common-modal'
+import { useUiNotifications } from '../../../../../notifications/ui-notification-boundary'
+import React, { useCallback } from 'react'
+import { Button, Modal } from 'react-bootstrap'
+import { Trans } from 'react-i18next'
+
+/**
+ * This modal for deletion of notes's revision
+ *
+ * @param show true to show the modal, false otherwise.
+ * @param onHide Callback that is fired when the modal is requested to close.
+ */
+export const RevisionDeleteModal: React.FC<ModalVisibilityProps> = ({ show, onHide }) => {
+  const { showErrorNotification } = useUiNotifications()
+  const noteId = useApplicationState((state) => state.noteDetails.id)
+
+  const deleteAllRevisions = useCallback(() => {
+    deleteRevisionsForNote(noteId).catch(showErrorNotification('editor.modal.deleteRevision.error')).finally(onHide)
+  }, [noteId, onHide, showErrorNotification])
+
+  return (
+    <CommonModal
+      show={show}
+      onHide={onHide}
+      titleI18nKey={''}
```

Comment:
Why is this empty?

I'd rather put the `editor.modal.deleteRevision.title` here and remove the `<h5>` in the body instead.

Reply:
done
![Screenshot 2023-07-24 at 10 43 05 AM](https://github.com/hedgedoc/hedgedoc/assets/26133749/c9ff7655-a8ed-4dfa-abb5-f1f85c0ed15a)